task/deprecated-image-check/0.1/README.md

```diff
@@ -0,0 +1,22 @@
+# deprecated-image-check task
+
+## Description:
+A check for deprecated images that are no longer maintained and will accumulate security issues.
+
+## Params:
+
+| name                | description                                         |
+|---------------------|-----------------------------------------------------|
+| POLICY_DIR          | Path to the directory containing conftest policies. |
+| POLICY_NAMESPACE    | Namespace for the conftest policy.                  |
+| BASE_IMAGES_DIGESTS | Digests of the base images used for build.          |
+
+## Params:
+
+| name              | description                               |
+|-------------------|-------------------------------------------|
+| PYXIS_HTTP_CODE   | HTTP code returned by Pyxis API endpoint. |
+| HACBS_TEST_OUTPUT | Test output.                              |
+
```

Comment:
I think it might be good to also add a link to the source repository, something like this:
```
## Source repository for image:
https://github.com/redhat-appstudio/hacbs-test
```